cli/integration-test/entrypoint.clj

```diff
@@ -65,7 +65,7 @@
 
   (apply require namespaces)
 
-  (let [test-results (timeout 600000
+  (let [test-results (timeout 800000
```

Comment:
I'm ok changing to that, but probably is not what is failing the tests but flakyness, check #704 for more details

Reply:
Yeah, it seems to sometimes take forever randomly on the diagnostics change integration test. I'll change it back then, no need to burn extra time if it's not going to finish anyway.